build_test_update.md

```diff
@@ -4,6 +4,8 @@ If you're looking to develop for TestGrid, welcome! Note that most of the inner
 are not open source yet. Follow [this issue](https://github.com/kubernetes/test-infra/issues/10409)
 for updates.
 
+:information_source: The Front End UI is not yet open source, if you would like UI changes or features, please file a bug describing how you would like the UI to behave. :information_source:
```

Comment:
nit: Using the word "yet" implies the team may have a specific timeline/deadline for open-sourcing the UI. Rephrasing it to something like "not currently open source" is probably more accurate at the moment (and won't be misleading to users).

Reply:
Done. Thought about that the second I submitted.